README.md

```diff
@@ -6,3 +6,4 @@ They lived in a thick, enchanting forest where the plants had grown tall and the
 Also, one of them had a very, very BIG secret; she could turn into a cupcake if she sticks out her tounge
 Anyways, back to the story…
 They were both going shopping one day then one of them got lost.
+One of them started to panic and accidently turned into a cupcake in the process.
```

Comment:
I love cupcakes. I would love to know what type of cupcake they turned into.

Reply:
They turned into a red velvet cupcake